index.js

```diff
@@ -67,6 +67,40 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener("click", e => {
+    toggleColor(e.target)
+})
+
+
 /***** Deliverable 2 *****/
+const likeButton = document.querySelector(".like-button")
+
+likeButton.addEventListener("click", () => {
+    traveler.likes += 1
+    likes.textContent = `${traveler.likes} Likes`
+})
 
 /***** Deliverable 3 *****/
+
+const form = document.querySelector("#new-animal-sighting-form")
+
+form.addEventListener("submit", (e) => {
+    e.preventDefault();
+    const species = form.species.value
+    const vidLink = form.link.value
+    const photoLink = form.photo.value
+    const description = form.description.value
+
+    const newSighting = {
```

Comment:
Looks good! RE: your second comment, do you mean something like this?

```javascript
const newData = {
        species,
        videoLink,
        photo,
        description
    }
```

if so, yes you could have done something like that! but i like this too - it's still clean and it's easy to understand.